tests/console/rust_cargo.pm

```diff
@@ -0,0 +1,84 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: cargo
+# Summary: Install cargo and assert full functionality
+#   - Create a new project
+#   - Add documentation
+#   - Add a dependency
+#   - Validate that the project can be run
+#   - Valdiate that documentation can be built by cargo
+# Maintainer: QE Core <qe-core@suse.de>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use utils qw(zypper_call);
+use Utils::Architectures 'is_aarch64';
+
+sub run {
+    select_console('root-console');
+    zypper_call('in cargo gcc');
```

Comment:
Why GCC? 

```suggestion
    zypper_call('in cargo');
```

Reply:
gcc technically is a cargo dependency afaik. It very much is for rustup anyway. Though it should be pre-installed I wanted to make sure.

Reply:
Nvm just checked with `zypper`. cargo itself only has rust as dependency. `gcc` is only required by `rustup`.